README.md

```diff
@@ -278,6 +278,7 @@ in the command line by using this command:
 ```sh
 cf login -a api.fr.cloud.gov --sso
 ```
+If you are offered a choice of orgs, select `gsa-tts-benefits-studio`. For the space, choose `notify-local-dev` to start with.
```

Comment:
These instructions are under Local Environment Setup → First-Time Project Setup. Checking the logs feels outside of that to me?

But, your point is taken that there should been guidance about how to target the space the user actually wants. Is that written anywhere I can link to? Otherwise, we might want to document it.

Reply:
I've added a parenthetical note that I hope clarifies this